[VTO-2962] Build packages for Ubuntu 18.04 and 20.04

mdippery opened the pull request:
Updates the build process to build package that are installable on Ubuntu 18.04 and 20.04.

mdippery committed:
Switch to Jenkins pipeline that supports Ubuntu 20.04

mdippery committed:
Build package on static-minion

The old nodes no longer exist.

mdippery committed:
[VTO-3039] Create Dockerfile to build package in Ubuntu 18.04

mdippery committed:
Explicitly configure names of Nexus repositories

mdippery committed:
[VTO-2962] Build package for Ubuntu 18.04

mdippery committed:
[VTO-3040] Build dlib package for Ubuntu 20.04

mdippery committed:
[VTO-3052] Reset jenkins-pipeline branch to release/v1.1

Necessary changes to support Ubuntu 20.04 have been merged into
release/v1.1.

tberkin commented on Dockerfile.bionic:
```diff
@@ -0,0 +1,56 @@
+# Standard issued requirements for all build enviornments in Ditto. We need
+# to maintain consistent sets of numerical libraries.
+
+RUN apt-get update -qq && apt-get install -y \
```
FYI, we removed some of these libraries from opencv build to remove the 3rd party dependencies. I think they might be existing here because we had a generic Dockerfile for all packages.
https://github.com/dittovto/ditto-opencv-builder/pull/11/files

mdippery replied:
Ah, okay. I see that the release/v3.3 was never merged into master, grrrrr. I'm going to submit a PR to merge that into master so we can pick up those changes.

mdippery replied:
Okay, I'll delete those packages from here, too.

mdippery replied:
I removed those packages and dlib builds fine. I'm guessing they weren't necessary and weren't linked anyway.

mdippery committed:
Remove unneeded dependencies

We have removed these for licensing reasons, but I don't think they're
necessary for building dlib anyway.
